src/conventions-and-definitions.md

```diff
@@ -81,3 +81,6 @@ The following assertions should all succeed:
         ]),
         10.0
     )
+
+It targets a normal distribution using [ziggurat](https://www.doornik.com/research/ziggurat.pdf) method.
+Note that implementation can diverge here. (TODO meaning using a different random function would be hard to detect and we don't attempt to).
```

Comment:
Implementations are _not_ allowed to diverge here. These random numbers must match between implementations or the delays computed by message senders will not match the real delays. This is why I've given a number of examples. Of course a malicious implementation can do whatever it wants, but that goes beyond just using a different random number generator.

I don't really want to go into detail on how the random number generation works here; I figured just saying it must match specific versions of the rand crates would be good enough? The rand library authors promise to provide deterministic portable random number generation where possible; see [here](https://rust-random.github.io/book/portability.html). If they change the algorithm in a new version and we are forced to use the new version, we may need to copy the old algorithm into the `mixnet` crate.